docs/samples/Microsoft.ML.Samples/Dynamic/SsaSpikeDetectorTransform.cs

```diff
@@ -27,36 +31,41 @@ class SsaSpikePrediction
 
         // This example creates a time series (list of Data with the i-th element corresponding to the i-th time slot). 
         // SsaSpikeDetector is applied then to identify spiking points in the series.
+        // SsaSpikeDetector differs from IidSpikeDetector in that it can account for temporal seasonality
+        // in the data.
         public static void SsaSpikeDetectorTransform()
         {
             // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
             // as well as the source of randomness.
             var ml = new MLContext();
 
-            // Generate sample series data with a spike
-            const int size = 16;
-            var data = new List<SsaSpikeData>(size);
-            for (int i = 0; i < size / 2; i++)
-                data.Add(new SsaSpikeData(5));
+            // Generate sample series data with a recurring pattern and a spike within the pattern
+            const int SeasonalitySize = 5;
+            const int TrainingSeasons = 3;
+            const int TrainingSize = SeasonalitySize * TrainingSeasons;
+            var data = new List<SsaSpikeData>();
+            for (int i = 0; i < TrainingSeasons; i++)
+                for (int j = 0; j < SeasonalitySize; j++)
+                    data.Add(new SsaSpikeData(j));
```

Comment:
Data generation used accross samples consider moving them in the Microsoft.ML.SamplesUtils.DatasetUtils.